### DISCUSSION TITLE
problem with the pauses created with  `. . .`  in revealjs slides

### DISCUSSION THREAD

**Discussion by perezp44** [quarto-dev/quarto-cli#5618] [Q&A] [answered]
Hello,
I have a question/problem about the pauses created with  `. . .`  in revealjs slides
In some cases (if a slide start with a header level higher than the slide-level) the pauses doesn't  work and you see the 3-dots in the final rendered html. I found a question related here (https://github.com/quarto-dev/quarto-cli/discussions/2300).

I think the problem comes for an interaction among the slide-level and the 3-dots; but I don't know if it is intentional or it is a bug.

Below there is an example with slide-level fixed to 2; in that case, the pauses created with the 3-dots works for slides that start with a second level header, but it doesn't work if you start a new slide with `---` and a three level header. In previous version of Quarto that worked for me, but not anymore with quarto 1.3.353

````qmd
---
title: "Problem with `. . .` to make pauses in rvealjs presentations"
date: 2023-05-21  
format: 
  revealjs:   
    slide-level: 2
---

## If the slide starts with `##` it works

Should be shown first

. . .

Should be shown latter

---

### If the slide starts with `###` it doesn't work

Should be shown first

. . .

Should be shown latter

````

**Comment by cderv** [comment, marked as answer]
> In previous version of Quarto that worked for me, but not anymore with quarto 1.3.353

This is a behavior of Pandoc that changed in [pandoc 3.0](https://pandoc.org/releases.html#pandoc-3.0-2023-01-18) that is now shipped with Quarto.

It seems that with the changed that was made to prevent fragment div to be created in non-slide format (https://github.com/jgm/pandoc/issues/8281), it is not possible to create pause in other type of headers

````markdown
---
title: "Problem with `. . .` to make pauses in rvealjs presentations"
date: 2023-05-21  
format: 
  revealjs:   
    slide-level: 2
---

## Slide Header

### If the pause is under a sub header, it doesn't work

Should be shown first

. . .

Should be shown latter
````

If you try to convert with Pandoc only, you'll get the `. . .` as is 

```powershell
quarto pandoc test2.qmd --from markdown --to revealjs --slide-level=2
```

This could be seen as a wrong fix for https://github.com/jgm/pandoc/issues/8281 in Pandoc

However not that you can create fragment content in slide directly for complex content like a sub header in slide 
See https://quarto.org/docs/presentations/revealjs/advanced.html#fragments

````markdown
---
title: "Problem with `. . .` to make pauses in rvealjs presentations"
date: 2023-05-21  
format: 
  revealjs:   
    slide-level: 2
---

## If the slide starts with `##` it works

Should be shown first

. . .

Should be shown latter

---

::: {.fragment}
### If the slide starts with `###` it doesn't work

Should be shown first
:::

::: {.fragment}
Should be shown latter
:::

````

Hope it helps

**Reply by perezp44** [reply]
@cderv thanks for the reply and for the hack with {.fragment}
I would prefer the previous behaviour but ... 
Thanks again

**Reply by cderv** [reply]
> I would prefer the previous behaviour but ...

You can open an issue in Pandoc repo directly to explain the change of behavior and see if this can be tweaked differently while still fixing https://github.com/jgm/pandoc/issues/8281

This is not something directly in Quarto. 

If you do open an issue there, do not hesitate to link here to it. thanks !

**Reply by rodrigo-j-goncalves** [reply]
I had the same problem and did not find the cause until I got here. 

Since this doesn't really depend on Quarto but on Pandoc, it would help if this is mentioned in the documentation when the 3-dots pause is described in the [incremental lists](https://quarto.org/docs/presentations/revealjs/#incremental-lists) section.

For example, change:
"_You can also insert a pause within a slide (keeping the content after the pause hidden) by inserting three dots separated by spaces:_" 
to
"_You can also insert a pause within a slide (keeping the content after the pause hidden) by inserting three dots separated by spaces. Note this only works on slides with header 1 (#) and/or header 2 (##):_"
